README_64bit.md

```diff
@@ -112,8 +112,17 @@ The flashing tool is known to be temperamental. Some steps you can try include:
 ### Device not showing up in MADmin -> System -> Auto-Config -> Pending Devices
 
 If at this point it has never showed up in MADmin and goes direct to flashing things in TWRP,
-then you have a problem with it reading the autoconfig file from your USB drive.
-You can try using the other USB port, be sure it is FAT formatted, no windows line endings.
+then you have a problem with it reading the autoconfig file from your USB drive, or the device
+has a problem accessing your MAD server due to incorrect settings or network issues.
```

Comment:
this is not true. if it found your autoconfig file but can not reach your server due to incorrect settings or network issues then it will not go direct to flashing in TWRP.